src/modules/database/database.js

```diff
@@ -1,3 +1,16 @@
-export let users = [];
+import pg from "pg";
```

Comment:
This file should not be in the modules folder.

Reply:
moved the database folder out of the modules folder